web/src/lib/auth/signinHelper.test.ts

```diff
@@ -1,5 +1,6 @@
 import * as api from "@/lib/api-client/apiClient";
 import { AuthAction } from "@/lib/auth/AuthContext";
+import * as session from "@/lib/auth/sessionHelper";
```

Comment:
dust - is this line actually needed? I don' see you using session. anything, but I might be misunderstanding how this works. Would love a bit of clarification

Reply:
This import was already in the file. I merely changed the way we imported it. And yes, we use the `getActiveUser` and `triggerSignOut` functions in `sessionHelper.ts` in the mock.